<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>EncodingDlgBase</class>
  <widget class="QDialog" name="EncodingDlg" >
    <property name="geometry" stdset="0" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>311</width>
        <height>180</height>
      </rect>
    </property>
    <property name="windowTitle" stdset="0" >
      <string>Choose default encoding</string>
    </property>
    <property name="sizeGripEnabled" stdset="0" >
      <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" stdset="0" >
        <number>11</number>
      </property>
      <property name="spacing" stdset="0" >
        <number>6</number>
      </property>
      <item>
        <widget class="QLabel" name="TextLabel1" >
          <property name="sizePolicy" stdset="0" >
            <sizepolicy>
              <hsizetype>1</hsizetype>
              <vsizetype>7</vsizetype>
            </sizepolicy>
          </property>
          <property name="text" stdset="0" >
            <string>Your current system encoding (UTF-8) may cause problems saving your personal information and reading offline messages from other users.</string>
          </property>
          <property name="alignment" stdset="0" >
            <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap" >
            <bool>true</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="TextLabel2" >
          <property name="text" stdset="0" >
            <string>You will choose the coding by default which want to use:</string>
          </property>
          <property name="wordWrap" >
            <bool>false</bool>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QComboBox" name="cmbEncoding" />
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" stdset="0" >
            <number>0</number>
          </property>
          <property name="spacing" stdset="0" >
            <number>6</number>
          </property>
          <item>
            <spacer name="Horizontal Spacing2" >
              <property name="sizeHint" >
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Qt::Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="QPushButton" name="buttonOk" >
              <property name="text" stdset="0" >
                <string>&amp;OK</string>
              </property>
              <property name="autoDefault" stdset="0" >
                <bool>true</bool>
              </property>
              <property name="default" stdset="0" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
          <item>
            <widget class="QPushButton" name="buttonCancel" >
              <property name="text" stdset="0" >
                <string>&amp;Cancel</string>
              </property>
              <property name="autoDefault" stdset="0" >
                <bool>true</bool>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>EncodingDlg</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>EncodingDlg</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
